doc/lua_api.md

```diff
@@ -6162,6 +6162,9 @@ Environment access
     * `objects`: if false, only nodes will be returned. Default is `true`.
     * `liquids`: if false, liquid nodes (`liquidtype ~= "none"`) won't be
                  returned. Default is `false`.
+    * `pointabilities`: Contains lists to override the `pointable` property of
```

Comment:
In the function description above, the `pointabilities` parameter is missing.
Should be
```md
 * `minetest.raycast(pos1, pos2, objects, liquids, pointabilities)`: returns `Raycast`
```

Reply:
Oops, thank you.